components/common/reporter/report_results.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Cisco Systems, Inc.
```

Comment:
- what is the output of this reporter? could you comment with an example output of it.
- this file should go to /components/tf_cnn_benchmarks/<filename.py>

Reply:
The output of this reporter is a result.csv file (the name of the output file is provided in this file examples/tf_cnn_benchmarks/test_workflow.yaml). Attaching the example image of the output (the actual .csv file cannot be attached for some reason).
<img width="514" alt="image" src="https://user-images.githubusercontent.com/9437376/40255946-b8ef5908-5a9d-11e8-827a-0763b62c469f.png">

As for the format - should it be something else considered/reordered?